src/libMessage/Messenger.cpp

```diff
@@ -493,6 +536,11 @@ bool ProtobufToAccountDelta(const ProtoAccount& protoAccount, Account& account,
     bool doInitContract = false;
 
     if (fullCopy) {
+      if (!CheckRequiredFieldsProtoAccountContract(protoAccount) ||
```

Comment:
Maybe we split the `if` and `LOG_GENERAL` for these two, so it will be easier for us to know which check failed